stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/EditReactionsView.kt

```diff
@@ -0,0 +1,87 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.ui.utils.UiUtils
+
+public class EditReactionsView : ReactionsView {
+    private val bubblePaint = Paint().apply {
```

Comment:
Why don't we have `ANTI_ALIAS_FLAG` on this `Paint`, like the ones in `ViewReactionsView`?